app/src/reducers/coverPage.js

```diff
@@ -0,0 +1,11 @@
+const initialState = null;
+import { GET_COVER_PAGE_ENTRIES } from '../actions';
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_COVER_PAGE_ENTRIES:
+      return action.payload;
```

Comment:
Ok. You're right on that. But we need to make sure that in case of any addition to this file, we need to change this line.

Thanks @nerik for the help.